compose/cli/main.py

```diff
@@ -565,7 +567,7 @@ def up(self, project, options):
         )
 
         if not detached:
-            log_printer = build_log_printer(to_attach, service_names, monochrome)
+            log_printer = build_log_printer(to_attach, service_names, monochrome, cascade_stop)
             attach_to_logs(project, log_printer, service_names, timeout)
```

Comment:
I think we need a check here before this condition to raise an error if this new flag is used with `-d`, since they don't work together, we shouldn't just ignore it in the `-d` case.